### PR TITLE
Adding low priority and variable pricing feature for standalone VMs. Added priority, eviction policy and billing profile to VirtualMachineProperties. Also, adding optional billing profile to VMSS Create and Update APIs.

### PR DESCRIPTION
**Pull request by rakeshakkera**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (yungezz) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: adding new property to stable version is better to be in a new service tag

Comment (rakeshakkera) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @yungezz CRP has implemented PUT as a Patch. This is an optional property, so not including the property will leave it the same as before.

**Comment by hyonholee** [comment]
LGTM

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json
Comment (rakeshakkera) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Please reopen if you have any further questions. Resolving conversation.

Comment (rakeshakkera) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Do you have any further questions @yungezz ?

**Comment by yungezz** [comment]
@rakeshakkera waiting for ARM feedback. Will ping ARM team.

**Comment by ravbhatnagar** [comment]
This change should ideally go in a new API-version as per the Microsoft REST API guidelines. But since CRP PUT is not compliant, I guess ARM can approve this sub-optimal modeling. @johngossman FYI and in case you have any input on this.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Review by hyonholee — APPROVED** [review]

**Comment by rakeshakkera** [comment]
> @rakeshakkera waiting for ARM feedback. Will ping ARM team.

@yungezz I have approval from ARM. Could you help merge the pull request?

**Review by huangpf — APPROVED** [review]
Approve on behalf of the team, since Hyonho has approved. 

P.S.
OneSDK team -- please help put me as an option review, if you see this. Thanks.

**Pull request merged**
